docs/dev/build-instructions/build_installer.md

```diff
@@ -2,11 +2,9 @@
 
 Currently, the installer creation has been tested on Windows only.
 
-For creating the installer, the [WiX Toolset](https://wixtoolset.org/) has to be installed on your system.
+For creating the installer, [WixSharp](https://github.com/oleg-shilo/wixsharp/releases/tag/v1.15.0.0) has to be present on your system. Please see the following [README](https://github.com/oleg-shilo/wixsharp/blob/master/README.md) for install information.
 
-An installer can be created after invoking cmake with the `-Dpackaging=ON` option and building.
+An installer can be created after invoking cmake with the `-Dpackaging=ON` option and building. This creates a *single-language* installer.
 
-To create a *single-language* installer (default English) run `cpack -C Release`.
-
-To create a *multi-language* installer run the script `scripts/Create-Win32InstallerMUI.ps1` from the root of this repository.
+To create a *multi-language* installer enable the following option `-Dtranslations=ON` and (re-)run the cmake configure step.
```

Comment:
This is actually not 100% true as `translations` is `ON` by default. Thus in order to create a _single-language_ installer, you'd explicitly have to use `-Dtranslations=OFF`.

----

Side note: I _personally_ prefer using underscores in markdown for making text italic (and stars to make it bold). Using the single stars to make text italic always raises the question (to me) whether the author might actually have wanted to create bold text but forgot a star :shrug:

Reply:
I will clean it up and move the `translations=OFF` option to the single language blurb. In regard to text formatting, I was mostly being consistent with the already present document formatting. If there is a better way or you would prefer we handle this at another time then I can remove it from the commit.

Reply:
Updated.

Reply:
Now that you mention it, I see that I was inconsistent with my style principles when writing that as well xD